client/src/components/PostBody.jsx

```diff
@@ -0,0 +1,25 @@
+
+
+function PostBody ({message}){
+
+    let arr = [];
+
+    function createChildren(message) {
+        let messageLines = message.split("\\n");
+        let messageImage = messageLines[messageLines.length-1].split(" ")[1];
```

Comment:
asi je to tu len pre ukazku, ale ked uz tak ten image by mal byt ako zvlast prop (tak ako message)